README.md

```diff
@@ -3400,6 +3400,48 @@ while (cond){
 }
 ```
 
+
+### requireNewlineAfterArrayElements
+
+Requires each element in array on a single line.
+
+Type: `Boolean` or `Number` (maximum) or `Object` (maximum: Infinity, ignoreBrackets: false)
+
+Values: `true`
+
+Default: Infinity
+
+#### Example
+
+```js
+"requireNewlineAfterArrayElement": 3
```

Comment:
id?

"Requires each element in array on a new line when size is more than passed number."

"Requires each element in array on a single line when length is bigger than `maximum`."

hm?